transcom/mymove
# B-17971 & B-17972 - Update TOO Submission To Prime Email

## Summary

Updating `move_issued_to_prime_template` with a new content provided by the gov.
This email gets sent whenever the TOO sends items to Prime either for Prime counseling move or Gov't counseled move.

Content of the email differs depending on Prime or Gov't counseling.

### How to test
Follow steps here to enable email in dev:
https://dp3.atlassian.net/wiki/spaces/MT/pages/1467973894/How+to+test+sending+email+locally

For Gov't counseling:

1. Use `HHGMoveNeedsSC` testharness to create the move and copy the locator code.
2. Login as counselor in KKFA Gblock -> find the move.
3. Scroll all the way down to Customer info and change the email to your email.
4. Submit the move as a Counselor and change role to TOO.
5. As a TOO, approve the move and you should receive an email.

For Prime counseling:

1. Create a new move as a marine corp, PPM move, and with NS, Mayport FL as all the duty location and use 32233 as your address zipcode(not sure if this matters).
2. Login as a TOO with USMC Gblock.
3. Find the move -> approve and you should receive an email.

## Screenshots
Gov't Email: 
![image](https://github.com/transcom/mymove/assets/146856854/27bafd38-def7-49a5-94e9-9bf9fd89b0f7)

Prime Email:
![image](https://github.com/transcom/mymove/assets/146856854/214a8893-bfde-44ff-9a7b-bcd6f5a83370)
